pkg/kratos/service.go

```diff
@@ -183,12 +183,20 @@ func (s *Service) CheckAllowedProvider(ctx context.Context, loginFlow *kClient.L
 	return s.contains(allowedProviders, fmt.Sprintf("%v", provider)), nil
 }
 
+func getClientName(loginFlow *kClient.LoginFlow) string {
```

Comment:
I would make it a method even if Service attributes are not used

it s simply a helper function so if it has to be separate then better to be part of the struct
other option is to keep it inline which leads to repetition

Reply:
Thanks, I'll go the method route!